package.json

```diff
@@ -39,7 +39,7 @@
     "opener": "^1.5.2",
     "prettier": "^2.0.5",
     "ts-jest": "^29.0.3",
-    "ts-node": "^9.1.1",
+    "ts-node": "^10.4.0",
```

Comment:
latest is 10.9.1, might as well try that and confirm that the behavior is still good?